### DISCUSSION TITLE
The kubelet is not running or unhealthy error after terraform apply

### DISCUSSION THREAD

**Discussion by umar-shabbir** [eclipse-theia/theia-cloud#272] [Q&A]
I am following Theia cloud [getting started instructions for minikube](https://github.com/eclipsesource/theia-cloud/blob/main/doc/docs/platforms/Minikube.md)
After running `terraform apply` it is stuck for a long time and prints following message to the console:
`module.cluster.minikube_cluster.cluster: Still creating...`

Then finally it is giving following error:
**This error is likely caused by:
│       - The kubelet is not running
│       - The kubelet is unhealthy due to a misconfiguration of the node in some way (required cgroups disabled)**

Any suggestions how to solve this issue?

Please see complete error message in the attached file.
[kubelet-error.txt](https://github.com/eclipsesource/theia-cloud/files/13977422/kubelet-error.txt)

**Additional Information:**
Platform: Windows 11
Kubernetes: 1.26.13
Minikube: 1.32.0
VirtualBox: 7.0.12

**Comment by jfaltermeier** [comment]
This seems to fail before Theia Cloud comes into play. 
Does creating a minikube cluster with `minikube start --vm=true --driver=virtualbox` work for you?
Which minikube version are you using? `minikube version`
If the cluster creation with above command works, please run `minikube dashboard` and check whether there are any failed pods in the "kube-system"-namespace (select namespace on top in toolbar)

**Reply by umar-shabbir** [reply]
@jfaltermeier thanks for the response. I tried the mentioned steps and it worked fine. There were no failed pods in the `kube-system` namespace
Also I had previously tried the [Hello Minikube](https://kubernetes.io/docs/tutorials/hello-minikube/) tutorial and it worked fine for me. But I could not succeed with Theia Cloud instructions for Minikube using Terraform

Minikube version 1.32.0
